app/src/main/java/com/sunasterisk/youtubebackground/screen/fragment/colectiontrending/CollectionTrendingFragment.kt

```diff
@@ -0,0 +1,87 @@
+package com.sunasterisk.youtubebackground.screen.fragment.colectiontrending
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.sunasterisk.youtubebackground.R
+import com.sunasterisk.youtubebackground.data.model.Page
+import com.sunasterisk.youtubebackground.data.source.VideoRepository
+import com.sunasterisk.youtubebackground.screen.adapter.VideoAdapter
+import com.sunasterisk.youtubebackground.utils.VideoCategoryId
+import kotlinx.android.synthetic.main.fragment_colection_trending.*
+
+class CollectionTrendingFragment : Fragment(), CollectionTrendingContract.View, View.OnClickListener{
+
+    private val adapter = VideoAdapter()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_colection_trending, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initData(null)
+        initView()
+        initButton()
+    }
+
+    private fun initButton() {
+        buttonImgTrending.setOnClickListener(this)
+        buttonImgGaming.setOnClickListener(this)
+        buttonImgMusic.setOnClickListener(this)
+    }
+
+    private fun initData(category: String?) {
```

Comment:
sử dụng giá trị default, để khi không truyền gì nó mặc định là null 
```suggestion
    private fun initData(category: String? = null) {
```